gradle.properties

```diff
@@ -21,3 +21,6 @@ android.useAndroidX=true
 
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
+
+# Used to set the maven version name to "local"
+com.thumbtack.android.overrideVersionNameWithLocal=false
```

Comment:
Hm, is there a cleaner way to do this to make it the default when publishing to the local maven repo? This approach is a bit less intuitive I think, and adds the chance of accidentally publishing a release with version name `local` if we forget to turn this back off. You can compare with https://github.com/thumbtack/android-lib-graphql/blob/master/lib/build.gradle.kts, which is a bit of a different situation since it doesn't have a default version name, only what's provided by CI.